core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJResolver.scala

```diff
@@ -15,6 +31,7 @@ case class IntelliJPatternResolver(pattern: String) extends IntelliJResolver {
       "orgPath" -> "com/jetbrains/intellij",
       "module" -> "idea",
       "artifact" -> artifact,
+      "ext" -> version.ext.get, // .get will be OK since we have `format = ".zip"` in the reference.conf file
```

Comment:
this comment is outdated, I think it is `ext = "zip"`. Make sure that dot is handled/expected consistently in all places.

Reply:
ok, let me fix the comment - in fact we have `ext = ".zip"` with a dot

Reply:
fixed as suggested